core/auth/anthropic_oauth.py

```
@@ -320,62 +320,40 @@ async def _refresh_claude_code_token(creds: dict[str, Any]) -> str | None:
+    # 1. Hexis-native PKCE OAuth (~/.hexis/auth/oauth.anthropic.json)
     hexis_creds = load_credentials()
     if hexis_creds:
         if not needs_refresh(hexis_creds.expires_ms, 120):
             return hexis_creds.access, "setup-token"
-        # Try to refresh
+        # Token is near expiry — refresh it in place.
         try:
             refreshed = await refresh_anthropic_token(hexis_creds.refresh)
             save_credentials(refreshed)
             return refreshed.access, "setup-token"
         except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)